ex03.exs

```diff
@@ -59,8 +59,32 @@ defmodule Ex03 do
 
   """
 
-  def pmap(collection, process_count, function) do
-    « your code here »
```

Comment:
## +4

This is a really nice catch. It's also a great example of why property-based testing is sometimes better than conventional unit tests.